engines/helm/app/models/k8s/cli.rb

```diff
@@ -16,13 +16,19 @@ def command()
       'kubectl'
     end
 
+    def prepare()
+      execute('version')
+    rescue CliError => e
+      # let kuberlr do it's thing
```

Comment:
s/it's/its ?

Reply:
Fixed in [d183ba3](https://github.com/SUSE-Enceladus/suse-rancher-setup/pull/206/commits/d183ba3c8517008db8363fb1ee647b325f5bd8e2)